js/views/pageNavVw.js

```diff
@@ -93,6 +85,23 @@ module.exports = Backbone.View.extend({
     this.notifcationSound.setAttribute('src', './audio/notification.mp3');
 
     this.render();
+
+    // pre-select lauguage and scroll it to the top.
+    var localLanguage = window.navigator.language;
+    var localLanguageFound = false;
+    var languageList = this.languages.get('languages');
+    for(var i in languageList) {
+      if(languageList[i].langCode == localLanguage) {
+        localLanguageFound = true;
+        break;
+      }
+    }
+    if(localLanguageFound == true) {
+      this.model.set('language', localLanguage);
+    }
+    else { // If local language is not defined, use en-US as default.
+      this.model.set('language', "en-US");
+    }
```

Comment:
This block should happen before render, so the language is set in the model for the first time the render happens.